extensions/unicode-symbols/src/components/CharacterActionPanel.tsx

```diff
@@ -19,8 +20,9 @@ export const CharacterActionPanel = ({ item }: { item: Character }) => {
     <ActionPanel>
       <ActionPanel.Section title="Unicode">
         <Action.Paste
-          title="Paste Character in Active App"
+          title={`Paste Article to ${frontmostApp?.name || "Active App"}`}
```

Comment:
Why is this renamed to 'Article'? This should be Character.

Reply:
Oh, sorry! It was a typo. I'm fixing now. Thanks for flagging.

Reply:
Done.